monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -682,6 +682,23 @@ object Observable {
   def cons[A](head: A, tail: Observable[A]): Observable[A] =
     new builders.ConsObservable[A](head, tail)
 
+  /** Creates a new observable from this observable and another given
+    * observable by interleaving their items into a strictly alternating sequence.
+    *
+    * So the first item emitted by the new observable will be the item whichever
+    * child stream emits first, and the second item will be emitted by the other
+    * child stream which did not get a chance to emit in the first place, the stream
+    * that did get the chance to emit the first item will wait until the other
+    * stream emits; this will continue until one of the child streams finishes,
+    * after that, the other stream will just keep emitting its items until
+    * `onComplete` is called.
+    *
+    * See [[merge]] for a more relaxed alternative that doesn't
+    * emit items in strict alternating sequence.
+    */
+  def interleave[A](oa1: Observable[A], oa2: Observable[A]): Observable[A] =
```

Comment:
@leakyabstraction dude, I jsut realized, this is not the place for the definitions of these operators.

Move them to `ObservableLike.scala` (and note that the operators are more or less in alphabetical order).
This is because these operators are also needed and specialized for `Subject` and for `Pipe`.